lib/generators/shopify_app/shop_model/templates/shop_helper.rb

```diff
@@ -0,0 +1,7 @@
+
+  protected
+
+  def shop
+    @shop ||= Shop.find_by(id: session[:shopify])
+  end
+  helper_method :shop
```

Comment:
what do you think about `current_shop` instead? This is how most the apps I work on have done it and it matches Devise which people may be familiar with.